### PR TITLE
[PWD-2504] - Finalize Homepage

### PR DESCRIPTION
**Pull request by ansorudin**
### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [X] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

**Review by haikal1223 — APPROVED** [review]

File: src/views/home/components/HistoryWatchVideoCard.tsx [hunk 1]
```diff
@@ -60,7 +60,11 @@ const HistoryWatchVideoCard: FC<HistoryWatchVideoCardProps> = ({
           </Typography>
         </Grid>
         <Grid item xs={2} display="flex" justifyContent="flex-end">
-          <Link to={`/dashboard/watch/${onlineCourseId}/topic/${topicId}`}>
+          <Link
```
Comment (FahemHakikiKhaya) [inline src/views/home/components/HistoryWatchVideoCard.tsx:63]: use PWDKLINK made by the swagest person @hiljaker biar gada styling bawaan

Comment (ansorudin) [reply src/views/home/components/HistoryWatchVideoCard.tsx:63]: notedd

File: src/views/home/components/WatchVideoCard.tsx [hunk 2]
```diff
@@ -77,7 +77,11 @@ const WatchVideoCard: FC<WatchVideoCardProps> = ({
           <Typography variant="paragraph">09:00 - 16:00</Typography>
         </Stack>
       </Stack>
-      <Link to={`/dashboard/watch/${onlineCourseId}/topic/${topicId}`}>
+      <Link
+        to={`/dashboard/watch/${onlineCourseId}/topic/${topicId}`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
```
Comment (shadiqfauzi) [inline src/views/home/components/WatchVideoCard.tsx:84]: yang ini perlu PWDKLink jg ga?

Comment (ansorudin) [reply src/views/home/components/WatchVideoCard.tsx:84]: ohh baik

**Review by shadiqfauzi — APPROVED** [review]
gaskeunn

**Review by Vesals — APPROVED** [review]

**Pull request merged**